cmd/psqldef/psqldef_test.go

```diff
@@ -590,6 +590,9 @@ func TestPsqldefCreateMaterializedView(t *testing.T) {
 			assertApplyOutput(t, createUsers+createPosts+createMaterializedView, applyPrefix+
 				fmt.Sprintf("CREATE MATERIALIZED VIEW %s.view_user_posts AS SELECT p.id FROM (%s as p JOIN %s as u ON ((p.user_id = u.id)));\n", tc.Schema, posts, users))
 			assertApplyOutput(t, createUsers+createPosts+createMaterializedView, nothingModified)
+
+			assertApplyOutput(t, createUsers+createPosts, applyPrefix+fmt.Sprintf(`DROP MATERIALIZED VIEW "%s"."view_user_posts";`, tc.Schema)+"\n")
+			assertApplyOutput(t, createUsers+createPosts, nothingModified)
```

Comment:
Thanks for adding a test. As noted in [this comment](https://github.com/sqldef/sqldef/blob/9cd896a2e1930968655efa208c100d6e6411e1b2/cmd/psqldef/psqldef_test.go#L65), we're actively migrating existing Go-based tests in `psqldef_test.go` to YAML-based tests in [tests.yml](https://github.com/sqldef/sqldef/blob/9cd896a2e1930968655efa208c100d6e6411e1b2/cmd/psqldef/tests.yml). We try to add new tests in `tests.yml` unless it's a CLI test.

Would you mind adding the new test to `tests.yml` instead (and migrating `TestPsqldefCreateMaterializedView` to `tests.yml` if possible)?

Reply:
I added test to the yaml. 
[0d80387](https://github.com/sqldef/sqldef/pull/499/commits/0d803870a2b141142fd94ca1172c9129e71ea45a)